### ISSUE TITLE
TiFlash queries kept timeout during TiUP applying patches, 

### ISSUE THREAD

**Issue by moyun** [pingcap/tiup#1414]
## Bug Report

1. What did you do?
User applied a patch in a TiDB 5.01 cluster (including 27 TiFlash nodes)
![tiup](https://user-images.githubusercontent.com/2842195/121112780-dd8c6480-c84b-11eb-8398-a97b4712006b.jpeg)
, and during the patching process TiFlash queries were not available/keeping throwing timeout errors. 
![tiflash-timeout](https://user-images.githubusercontent.com/2842195/121112788-e1b88200-c84b-11eb-969f-0b373e922a4c.jpeg)

2. What did you expect to see?
3. What did you see instead?
Retry of queries should be executed normally during the patching. 
Even if a query was interrupted when a single node was applying the patch but it should have a chance to be executed in an interval between applying patches of two nodes, but not kept being interrupted.  
There's no way for users to adjust the interval of applying patch between two different nodes.

4. What version of TiUP are you using (`tiup --version`)?
1.3.7

**Comment by AstroProfundis** [comment]
I think this should be an issue of TiFlash or PD.

**Comment by moyun** [comment]
yes, perhap TiFlash or PD have their issues but can TiUp give an option that let users can adjust the interval between two restarts? This may more or less help to avoid or reduce the chance of the failures.
